cranelift/codegen/src/isa/x64/inst/mod.rs

```diff
@@ -489,6 +488,70 @@ pub(crate) fn low32_will_sign_extend_to_64(x: u64) -> bool {
     xs == ((xs << 32) >> 32)
 }
 
+impl Inst {
+    fn iset_requirement(&self) -> Option<InstructionSet> {
```

Comment:
Could we rename this to `isa_requirement()`? I think "ISA" is a more standard term than "iset" as an abbreviation.